snooty/parser.py

```diff
@@ -1059,6 +1059,12 @@ def _locate_text(text: str) -> int:
             elif image_argument:
                 self.validate_and_add_asset(doc, image_argument, line)
 
+        elif key in {"mongodb:collapsible"}:
```

Comment:
this should just be `key == "mongodb:collapsible"`, otherwise we're creating a set object just for a __contains__ call

Reply:
good catch. updating